incuna_test_utils/testcases/api_examples.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+
+class APIExampleMixin:
+    """A TestCase mixin that allows access to API by Example retrieval methods."""
```

Comment:
Oh ignore me, just saw your base test subclass.  Bear in mind that this mixin is not particularly usable without that.